test.py

```diff
@@ -946,7 +942,7 @@ def _prepare_pytest_params(self, options: argparse.Namespace):
             # https://docs.pytest.org/en/7.1.x/reference/exit-codes.html
             no_tests_selected_exit_code = 5
             self.valid_exit_codes = [0, no_tests_selected_exit_code]
-        self.args.append(str(self.suite.suite_path / (self.name + ".py")))
+        self.args.append(str(self.suite.suite_path / self.name))
```

Comment:
Adds extra painful typing :/ and makes it hard to remember. @nyh @kbr-scylla @denesb do you really think that compatibility with dtest/pytest should lead to so much extra typing?

Reply:
I think the ".py" can be optional - but if someone types it (because they are familiar with pytest, perhaps), what does it hurt to allow it as well?